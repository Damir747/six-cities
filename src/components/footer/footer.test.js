import React from 'react';
import { render } from '@testing-library/react';
import { Router } from 'react-router-dom/cjs/react-router-dom.min';
import { createMemoryHistory } from 'history';
import Footer from './footer';

it(`Should Footer render correctly`, () => {
  const history = createMemoryHistory();
  const { getByTestId, getByAltText } = render(
    < Router history={history}>
      <Footer />
    </ Router>
  );
  const footerLogo = getByTestId('footer__logo');
  const footerLogoAlt = getByAltText('6 cities logo');
  expect(footerLogo).toMatchSnapshot();
  expect(footerLogoAlt).toMatchSnapshot();
});
