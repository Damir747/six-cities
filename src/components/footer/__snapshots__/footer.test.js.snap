// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Footer render correctly 1`] = `
<img
  alt="6 cities logo"
  class="footer__logo"
  data-testid="footer__logo"
  height="33"
  src="img/logo.svg"
  width="64"
/>
`;

exports[`Should Footer render correctly 2`] = `
<img
  alt="6 cities logo"
  class="footer__logo"
  data-testid="footer__logo"
  height="33"
  src="img/logo.svg"
  width="64"
/>
`;
